src/citrine/_rest/engine_resource.py

```diff
@@ -55,38 +54,6 @@ def is_archived(self):
         """:bool: whether the resource is archived (hidden but not deleted)."""
         return self.archived_by is not None
 
-    @property
-    @deprecated(deprecated_in="1.46.0", removed_in="2.0.0",
-                details="Please use the 'status_detail' property instead.'")
-    def status_info(self):
-        """[DEPRECATED] :list[str]: human-readable explanations of the status."""
-        return [detail.msg for detail in self.status_detail]
-
-    @property
-    @deprecated(deprecated_in="1.31.0", removed_in="2.0.0",
-                details="Please use the 'is_archived' property instead.'")
-    def archived(self):
-        """[DEPRECATED] whether the resource is archived."""
-        return self.is_archived
```

Comment:
Note that design spaces, which are modules and not engine resources, still use `.archived`